org.entangle_photo.Manager.json

```diff
@@ -0,0 +1,131 @@
+{
+    "app-id": "org.entangle_photo.Manager",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "entangle",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "libghoto2",
+            "cleanup" : ["/bin", "/lib/udev", "/share/doc"],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://downloads.sourceforge.net/project/gphoto/libgphoto/2.5.23/libgphoto2-2.5.23.tar.bz2",
+                    "sha256" : "d8af23364aa40fd8607f7e073df74e7ace05582f4ba13f1724d12d3c97e8852d"
+                }
+            ],
+            "config-opts" : ["--disable-introspection", "--disable-docs"]
+        },
+        {
+            "name": "gudev",
+            "config-opts" : ["--disable-umockdev"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libgudev/232/libgudev-232.tar.xz",
+                    "sha256": "ee4cb2b9c573cdf354f6ed744f01b111d4b5bed3503ffa956cefff50489c7860"
+                }
+            ]
+        },
+        {
+            "name": "intltool",
```

Comment:
To avoid duplicates, please use intltool from shared-modules.

Reply:
Is not needed anymore.